client/src/main/java/alcatraz/client/Main.java

```diff
@@ -86,16 +85,17 @@ public static void main(String[] args) {
                             case "1":
                                 /* Start Game */
                                 ArrayList<Player> lobbyPlayers = server.initializeGameStart(lobby.lobbyId, lobby.secret);
-                                for(int i = 1; i < lobbyPlayers.size(); i++){
+                                for(int i = 0; i < lobbyPlayers.size(); i++){
                                     try {
                                         if(!lobbyPlayers.get(i).getClientName().equals(clientName)){
                                             lobbyPlayers.get(i).getClient().startGame(lobbyPlayers, i);
+                                        }else{
+                                            lobbyPlayers.get(i).getClient().startGame(lobbyPlayers, 0);
```

Comment:
i think this is not necesarry, since lobbyPlayer[0] will always be the owner of the lobby... looping through every sould work fine